appcelerator/titanium_mobile
# chore(android): remove hidden API usage

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-27743

**Test:**
1. Create a default Classic Titanium app project.
2. Build and run on Android 11.
3. Verify the following 2 warning messages are **NOT** logged.
```
Accessing hidden method Landroid/content/res/Resources;->getCompatibilityInfo()Landroid/content/res/CompatibilityInfo; (light greylist, reflection)
Accessing hidden field Landroid/content/res/CompatibilityInfo;->applicationScale:F (greylist, reflection, allowed)
```

_**Note**: Other `Accessing hidden method...` warnings may appear, but we can't prevent them since they come from Google's libraries and not our code._
